bundlegen/schema/appMetadataSchema.json

```diff
@@ -0,0 +1,70 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
```

Comment:
can we please use the latest JSON-schema spec which is 2020-12:
https://json-schema.org/specification.html

Reply:
@goruklu - updated the schema specs as per 2020-12 in the latest commit.